src/graphs.jl

```diff
@@ -365,13 +371,35 @@ end
                     _3d && push!(zseg, z[si], z[di], NaN)
                     push!(l_wg, wi)
                 end
+            if typeof(edgelabel) == Dict
+                if edge_label_exsists(edgelabel, (si, di))
```

Comment:
typo shound be  `exists`

Reply:
Wow, that is embarassing. I am really bad without a spellchecker.